.github/workflows/build.yml

```diff
@@ -463,6 +463,64 @@ jobs:
           name: 'PackSquash CLI AppImage zsync (arm64)'
           path: PackSquash-*-aarch64.AppImage.zsync
 
+  docker-image:
+
+    runs-on: ubuntu-latest
+
+    env:
+      # Use docker.io for Docker Hub if empty
+      REGISTRY: ghcr.io
+      # github.repository as <account>/<repo>
+      IMAGE_NAME: ${{ github.repository }}
+
+    needs: [ linux-x64-gnu ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Docker image metadata
+        id: dockermeta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          # generate Docker tags: edge for every commit on master, semver for every semantic tag
+          tags: |
+            type=edge,branch=master
+            type=semver,pattern={{version}}
+        
+      - name: Download PackSquash AppImage
+        uses: actions/download-artifact@v3.0.0
+        with:
+          name: 'PackSquash CLI AppImage (x64)'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
Is it necessary to set up QEMU here? To me, it doesn't look like it.

And now that I see this, did you try to build Docker images for ARM64/AArch64 too? Docker supports that architecture, PackSquash supports it too, and some advanced users will appreciate it.

Reply:
![изображение](https://user-images.githubusercontent.com/20980281/168444568-848059ea-ce39-4d3d-a716-a0966e07e370.png)
I believe arm64 does build, though aarch64 would be a challenge since you have a separate binary for it

Reply:
Okay about QEMU: it was not necessary, but if we do multi-platform images, we would need it

Reply:
Yes, it might be necessary to test the image, although doing so is a bit complex. The PackSquash binaries built by the workflow are tested on ARM64, though, so if the Docker image is largely the same between architectures, barring the different AppImage, we can assume it works fine on both.

Reply:
@AlexTMjugador QEMU is not necassary